cupy/fft/fft.py

```diff
@@ -420,7 +423,10 @@ def _default_plan_type(a, s=None, axes=None):
     return 'nd'
 
 
-def _default_fft_func(a, s=None, axes=None):
+def _default_fft_func(a, s=None, axes=None, plan=None):
+    if plan is not None: # a shortcut for using _fftn
```

Comment:
Maybe
```Python
if isinstance(plan, 'cufft.PlanNd'):
```
to be a bit more specific? We may want to allow passing `Plan1D` plans as well at some point.

Reply:
Good point. Done.